server/sonar-server/src/test/java/org/sonar/server/organization/ws/SearchMyOrganizationsActionTest.java

```diff
@@ -159,8 +161,8 @@ public void returns_organization_of_authenticated_user_only_for_ADMIN_permission
     OrganizationDto organization2 = dbTester.organizations().insert();
     GroupDto group = dbTester.users().insertGroup(organization2);
     dbTester.users().insertMember(group, user);
-    GlobalPermissions.ALL.stream()
-      .filter(s -> !s.equals(SYSTEM_ADMIN))
+    Arrays.stream(OrganizationPermission.values())
```

Comment:
could be worth having a method `OrganizationPermission.all()` which returns a `Stream<OrganizationPermission>`, WDYT?

`GlobalPermissions.ALL` was used in several places